resources/wikia/modules/facebookLocale.js

```diff
@@ -0,0 +1,286 @@
+				'ES': ['ca', 'es', 'eu', 'gl'],
+				'ET': ['am'],
+				'FI': ['fi'],
+				'FR': ['br', 'co', 'fr'],
+				'FO': ['fo'],
+				'GB': ['cy', 'en'],
+				'GE': ['ka'],
+				'GH': ['ak'],
+				'GR': ['el', 'gx'],
+				'HK': ['zh'],
+				'HR': ['hr'],
+				'HU': ['hu'],
+				'ID': ['id', 'jv'],
+				'IE': ['ga'],
+				'IL': ['he'],
+				'IN': ['as', 'bn', 'en', 'gu', 'hi', 'kn', 'ml', 'mr', 'or', 'pa', 'sa', 'ta', 'te'],
+				'IQ': ['cb'],
+				'IR': ['fa'],
+				'IS': ['is'],
+				'IT': ['it', 'sc'],
+				'JP': ['ja'],
+				'KE': ['sw'],
+				'KH': ['km'],
+				'KR': ['ko'],
+				'KS': ['ja'],
+				'KZ': ['kk'],
+				'LA': ['es', 'lo'],
+				'LK': ['si'],
+				'LT': ['fb', 'lt'],
+				'LV': ['lv'],
+				'MA': ['tz'],
+				'MG': ['mg'],
+				'MK': ['mk'],
+				'MM': ['my'],
+				'MN': ['mn'],
+				'MT': ['mt'],
+				'MW': ['ny'],
+				'MX': ['es'],
+				'MY': ['ms'],
+				'NG': ['ff', 'ha', 'ig', 'yo'],
+				'NL': ['fy', 'li', 'nl'],
+				'NO': ['nb', 'nn', 'se'],
+				'NP': ['ne'],
+				'PE': ['qu'],
+				'PH': ['cx', 'tl'],
+				'PI': ['en'],
+				'PK': ['ur'],
+				'PL': ['pl', 'sz'],
+				'PT': ['pt'],
+				'PY': ['gn'],
+				'RO': ['ro'],
+				'RS': ['sr'],
+				'RU': ['ru', 'tt'],
+				'RW': ['rw'],
+				'SE': ['sv'],
+				'SI': ['sl'],
+				'SK': ['sk'],
+				'SN': ['wo'],
+				'SO': ['so'],
+				'ST': ['tl'],
+				'SY': ['sy'],
+				'TH': ['th'],
+				'TJ': ['tg'],
+				'TM': ['tk'],
+				'TR': ['ku', 'tr', 'zz'],
+				'TW': ['zh'],
+				'UA': ['uk'],
+				'UD': ['en'],
+				'UG': ['lg'],
+				'US': ['ck', 'en'],
+				'UZ': ['uz'],
+				'VA': ['la'],
+				'VE': ['es'],
+				'VN': ['vi'],
+				'ZA': ['af', 'xh', 'zu'],
+				'ZW': ['nd', 'sn']
+			};
+
+		/**
+		 * Gets the Localized URL of the FB JS SDK
+		 * To localize, it attempts to match user location (country) with the language;
+		 * if unsuccessful, it falls back to a default country associated with the language
+		 * In case language is invalid or unsupported, defaults to en/US
+		 *
+		 * @param {string} inputLanguageCode
+		 * @returns {string} URL of localized Facebook SDK
+		 */
+		function getSdkUrl(inputLanguageCode) {
+			var matchingLanguages,
+				languageCode = inputLanguageCode.length === 2 ? inputLanguageCode.toLowerCase() : 'en',
+				geoCountryCode = Geo.getCountryCode(),
+				countryCode = '';
+			if (geoCountryCode) {
+				matchingLanguages = languagesOfCountry[geoCountryCode.toUpperCase()];
+				if (matchingLanguages && languageCode in matchingLanguages) {
+					countryCode = geoCountryCode.toUpperCase();
+				}
+			}
+			if (!countryCode) {
+				countryCode = defaultCountryCodes[languageCode];
+				if (!countryCode) {
+					languageCode = 'en';
+					countryCode = 'US';
+				}
+			}
+
+			return '//connect.facebook.net/' + languageCode + '_' + countryCode + '/sdk.js';
+		}
+
+		return {
+			getSdkUrl: getSdkUrl
+		};
+	}
+
+	//UMD inclusive
+	if (!context.Wikia) {
+		context.Wikia = {};
+	}
+
+	//namespace
+	context.Wikia.fbLocale = facebookLocale(context);
```

Comment:
It doesn't look like `facebookLocale` takes any arguments.  Is there something else going on here I'm not seeing?